keyboard/src/lib/y2country/widgets.rb

```diff
@@ -0,0 +1,79 @@
+# encoding: utf-8
+
+# ------------------------------------------------------------------------------
+# Copyright (c) 2016 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+#
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of version 2 of the GNU General Public License as published by the
+# Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may find
+# current contact information at www.novell.com.
+# ------------------------------------------------------------------------------
+
+require "yast"
+require "cwm/widget"
+
+Yast.import "Keyboard"
+
+module Y2Country
+  module Widgets
+    class KeyboardSelectionWidget < CWM::SelectionBox
```

Comment:
[The scheme](https://gist.github.com/imobachgs/d90940315bd0471ee00b2a68ba22fbe5#file-proposal-md) is not actually clear on this. but as I understood it, this class should be named `Y2Country::Widgets::KeyboardSelection`, not `Y2Country::Widgets::KeyboardSelectionWidget`.
@imobachgs we should make that clear.